hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml

```diff
@@ -212,7 +212,7 @@
 
 - project:
     name: kubernetes-e2e-gce-1-2
```

Comment:
Done. Doesn't really matter, but seems good for consistency. I didn't touch the Trusty jobs.